ingesters/winevent.md

```diff
@@ -136,13 +163,26 @@ Restart the Gravwell service via standard windows service management.
         Channel=Microsoft-Windows-Sysmon/Operational
 ```
 
-## Troubleshooting
+## Windows Event Forwarding
 
-You can verify the Windows ingester connectivity by navigating to the Ingester page on the web interface.  If the Windows ingester is not present, check the status of the service either via the windows GUI or by running `sc query GravwellEvents` at the command line.
+The Gravwell Winevent ingester can be combined with Windows Event Forwarding (WEF) to simplify deployments and reduce the number of endpoints the ingester must be installed on.  Windows Event Forwarding is an integrated Windows service that allows for forwarding events to a central collection point using integrated Windows functionality.  More information on WEF can be found on [several](https://learn.microsoft.com/en-us/windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection) [Microsoft](https://social.technet.microsoft.com/wiki/contents/articles/33895.windows-event-forwarding-survival-guide.aspx) [resources](https://learn.microsoft.com/en-us/defender-for-identity/configure-event-forwarding).
 
-![](querystatus.png)
+Configuring Windows Event Forwarding is beyond the scope of this document, but actually collecting the forwarded events is very simple.
 
-![](querystatusgui.png)
+First you will need to install the winevent ingester on the Windows system that will be receiving the forwarded events.  Then you will want to validate the name of the channel that is configured to receive the forwarded events on the collection box.  Typically that is ForwardedEvents`.
```

Comment:
You're missing a backtick at the start of ForwardedEvents and it's messing things up elsewhere.